doc/release-notes.md

```diff
@@ -1,3 +1,18 @@
+# Release 0.15.1
+
+* What's New:
```

Comment:
I'd probably say 'No new functionality introduced'. Can you use functionality as an adjective?

Reply:
Changed. Also I'm not sure.